web/src/containers/activity/ContractorResource/ContractorResourceReview.test.js

```diff
@@ -37,13 +37,17 @@ describe('the ContractorResourceReview component', () => {
   };
 
   it('renders correctly', () => {
-    const component = shallow(<ContractorReview {...props} />);
+    const component = shallow(
+      <ContractorReview {...props}>
+        <div/>
```

Comment:
@radavis If you remember, why is this child element necessary here? As far as I can tell, the contractor review component doesn't require any children, but I may just be missing something.

Reply:
I think it is because the Review component, which is being used by the ContractorReview component, has `props.children` marked as `isRequired`